README.md

```diff
@@ -122,6 +122,14 @@ command line tool:
 * In case of irrecoverable errors, such as detected schema changes without an
   upgrade path, the tables and their indexes may be dropped with `rebuilddb2 -D`.
 
+Note that dcrdata requires that your [dcrd RPC](https://docs.decred.org/getting-started/user-guides/dcrd-setup/) is running with optional indexes enabled.  By default these indexes are not turned on when dcrd is installed.
```

Comment:
the term "dcrd RPC" is a bit confusing.  It would be fine to say just "...that dcrd is running with..."

Reply:
done!